includes/class-pb-globaltypography.php

```diff
@@ -20,6 +20,7 @@ function getSupportedLanguages() {
 			'grc' => __( 'Ancient Greek', 'pressbooks' ),
 			'ar' => __( 'Arabic', 'pressbooks' ),
 			'he' => __( 'Biblical Hebrew', 'pressbooks' ),
+			'cre' => __( 'Canadian Indigenous ', 'pressbooks' ),
```

Comment:
My convention has been to use the ISO 639-1 identifier for languages, but since this  is an instance where the name of the script and the languages are not identical and where the script supports multiple languages (many of which are not Cree variants), I'd like to request that you switch from `cre` to the ISO 15924 script name identifier, `cans` (see: https://en.wikipedia.org/wiki/ISO_15924).

Also in keeping with this, could you modify the description to Canadian Indigenous Syllabics to be more in line with the  ISO 15924 name? Thanks @bdolor!